packages/cozy-harvest-lib/src/helpers/queries.js

```diff
@@ -1,15 +1,49 @@
 import CozyClient, { Q } from 'cozy-client'
 
-const DEFAULT_CACHE_TIMEOUT_QUERIES = 9 * 60 * 1000
+const DEFAULT_CACHE_TIMEOUT_QUERIES = 10 * 60 * 1000
 const defaultFetchPolicy = CozyClient.fetchPolicies.olderThan(
   DEFAULT_CACHE_TIMEOUT_QUERIES
 )
 
-export const buildKonnectorQuery = slug => ({
-  definition: () =>
-    Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+export const buildKonnectorQuery = (slug, source) => {
+  const query = {
+    definition: () =>
+      Q('io.cozy.konnectors').getById(`io.cozy.konnectors/${slug}`),
+    options: {
+      as: `konnector-${slug}-${source}`,
+      fetchPolicy: defaultFetchPolicy
+    }
+  }
+  const queryDef = query.definition()
+  queryDef.sources = [source]
+
+  return { definition: queryDef, options: query.options }
+}
+
+export const buildIsAccountConnectedQuery = sourceAccount => ({
+  definition: Q('io.cozy.accounts').getById(sourceAccount),
+  options: {
+    as: `isAccountConnected-${sourceAccount}`,
```

Comment:
I propose to name the queries retrieving only 1 doc into `<doctype>/<id>`. In the future, we could make it a default inside cozy-client (automatic naming for getById queries).

Reply:
J'ai pas compris comment tu proposes de renommer `buildIsAccountConnectedQuery` ... ?

Reply:
Je propose de nommer différent la query : 

```
-     as: `isAccountConnected-${sourceAccount}`, 
+     as: `io.cozy.accounts/${sourceAccount}`,
```

L'idée étant que le doc associé à la query n'a pas à être scopée sous la fonctionnalité `isAccountConnected` vu que l'account pourrait être utilisé ailleurs, et on voudrait pouvoir se reposer sur le cache : si on pose comment convention que les Q().getById() sont nommées <doctype>/<id>, on peut partager le cache du document à plusieurs composants. Ici un autre  composant n'aurait pas idée d'utiliser la query `isAccountConnected-${sourceAccount}` pour récupérer son document.